### PR TITLE
ES5 Adds ARIA role to window and aria helper functions to widget

### PR DESCRIPTION
**Pull request by ArturWesner**
ES5 Version of [#10290](https://github.com/qooxdoo/qooxdoo/pull/10290)

### CODE REVIEW

File: source/class/qx/ui/core/Widget.js [hunk 1]
```diff
@@ -3827,6 +3827,76 @@ qx.Class.define("qx.ui.core.Widget",
...
+          idArr.push(widgetId);
+        }
+      });
+      if (idArr.length === 0) {
+        return;
+      }
+      var idStr = idArr.join(" ");
+      var contentEl = this.getContentElement();
+      var res = contentEl.getAttribute(ariaAttr);
+      res = res ? `${res} ${idStr}` : idStr;
```
Comment (derrell) [inline source/class/qx/ui/core/Widget.js:3895]: Template literals are not available in ES5.

**Review by derrell — CHANGES_REQUESTED** [review]

**Review by derrell — APPROVED** [review]

**Review by hkollmann — APPROVED** [review]

**Comment by oetiker** [comment]
@ArturWesner how about makeing this a PR to https://github.com/qooxdoo/qooxdoo/tree/latest directly ?

**Review by cboulanger — APPROVED** [review]

**Comment by ArturWesner** [comment]
@oetiker Yes, thats what i thought/meant too.
Oops, seems like changing the branch from master to latest broke the PR. I thought it would kinda copy the commits onto the latest branch. Hmm. What should i do now? Create a new PR and delete this one?
Sorry for the inconvenience!

**Comment by oetiker** [comment]
> @oetiker Yes, thats what i thought/meant too. Oops, seems like changing the branch from master to latest broke the PR. I thought it would kinda copy the commits onto the last commit of the latest branch. Hmm. What should i do now? Create a new PR and delete this one? Sorry for the inconvenience!

best is todo a rebase of your branch to latest and then push it `git fetch upstream; git rebase latest; git push --force`

**Comment by hkollmann** [comment]
What's about the earlier pr https://github.com/qooxdoo/qooxdoo/commit/68b240c5404eee3b28a4651030c4886e5509e6e5?

Could you make a PR for latest too?

**Review by oetiker — APPROVED** [review]

**Comment by oetiker** [comment]
> What's about the earlier pr [68b240c](https://github.com/qooxdoo/qooxdoo/commit/68b240c5404eee3b28a4651030c4886e5509e6e5)?
> 
> Could you make a PR for latest too?

this PR is for latest!

**Comment by ArturWesner** [comment]
In the mentioned PR `const` and `let`needs to be replaced by `var`. And of course the changes should be applied to ./framework/source instead of ./source. 
I can create an extra PR if needed. Is there an better way to apply the directory change than creating a patch with `git format-patch` and applying it with `git apply --directory='framework'` ?

**Comment by hkollmann** [comment]
just try it...

**Review by cboulanger — APPROVED** [review]

**Review by level420 — APPROVED** [review]

**Pull request merged**